src/common/arrow/Cargo.toml

```diff
@@ -1,10 +1,16 @@
 [package]
-authors = ["Databend Authors <opensource@datafuselabs.com>"]
+authors = [
```

Comment:
authors has been deprecated by cargo, we can remove this field.

Reply:
ok